Issue on "make -j" command with python3

There seems to be an error on the "make -j" command with Python3, due to cpplint.py.

`[...]
src/CMakeFiles/lint.pfaedle.dir/build.make:112: recipe for target 'src/CMakeFiles/lint.pfaedle' failed
make[2]: *** [src/CMakeFiles/lint.pfaedle] Error 1
CMakeFiles/Makefile2:172: recipe for target 'src/CMakeFiles/lint.pfaedle.dir/all' failed
make[1]: *** [src/CMakeFiles/lint.pfaedle.dir/all] Error 2
make[1]: *** Waiting for unfinished jobs....
Makefile:140: recipe for target 'all' failed
make: *** [all] Error 2
[...]
`

Tested with Debian9.4/Python3.6.5 and Ubuntu18.04~Debian10/Python3.6.2.
No problem with Python2.7.

Well cordially and thank you for your work.

--- a/cpplint.py
+++ b/cpplint.py
@@ -1,4 +1,4 @@
-#!/usr/bin/env python
+#!/usr/bin/env python2
 #
 # Copyright (c) 2009 Google Inc. All rights reserved.
 #

